addons/crm/models/crm_lead.py

```diff
@@ -1189,10 +1190,12 @@ def _track_subtype(self, init_values):
         self.ensure_one()
         if 'stage_id' in init_values and self.probability == 100 and self.stage_id:
             return self.env.ref('crm.mt_lead_won')
-        elif 'active' in init_values and self.probability == 0 and not self.active:
+        elif 'lost_reason' in init_values:
             return self.env.ref('crm.mt_lead_lost')
         elif 'stage_id' in init_values:
             return self.env.ref('crm.mt_lead_stage')
+        elif 'team_id' not in init_values and self.active:
```

Comment:
Why a team id based condition ?

Reply:
I don't know, this was done by jeh, I think it can be removed.